src/smif/convert/register.py

```diff
@@ -1,10 +1,36 @@
-"""Register and ResolutionSet abstract classes to contain area and interval
-metadata.
+"""Register, NDimensionalRegister and ResolutionSet
+abstract classes to contain area, interval and unit metadata.
+
+Implemented by :class:`smif.convert.interval.TimeIntervalRegister`,
+:class:`smif.convert.area.RegionRegister` and
+:class:`smif.convert.unit.UnitRegister`.
 """
+import logging
 from abc import ABCMeta, abstractmethod
+from collections import OrderedDict, defaultdict
+
+import numpy as np
+
+
+class LogMixin(object):
```

Comment:
Not sure about introducing the Mixin here for only the single property. Any disadvantage to including `logger` in `Register`?

Reply:
Using logger as an inherited instance variable prevented the logs from appearing in any outputs... this seems to work, and could be placed elsewhere (e.g. smif.__init__) hold all central logging settings